.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: android
+
+jdk:
+  - oraclejdk8
+
+android:
+  components:
+    - tools
+    - platform-tools
+    - build-tools-27.0.3
+    - android-27
+
+sudo: false
+
+before_install:
+  - touch local.properties
+  - yes | sdkmanager "platforms;android-27"
+
+script:
+  - ./gradlew --no-daemon clean build -Pcoverage
```

Comment:
Don't we need to run "jacocoTestReport" here too?

Reply:
Oh, I just noticed that we are running "build" instead of "assemble" here. "Build" seems to run a bunch of things including "lint". Maybe running jacocoTestReport explicitly would be faster?